.github/workflows/update-release-version.yml

```diff
@@ -0,0 +1,96 @@
+name: 'Update release version'
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        description: 'Release type'
+        required: true
+        default: 'preview' 
+        type: choice
+        options:
+        - rtm
+        - rc
+        - preview
+        - alpha
+      release_version:
```

Comment:
Can you validate that this is a 3-part version without any labels? Later parts in the script assume that the last part is a patch version, but that's not the case if some specifies `8.0`.